packages/docs-app/src/examples/core-examples/popoverDismissExample.tsx

```diff
@@ -36,6 +36,8 @@ export class PopoverDismissExample extends React.PureComponent<
         return (
             <Example options={false} {...this.props}>
                 <Popover
+                    autoFocus={false}
+                    enforceFocus={false}
```

Comment:
This Popover example continuously reopens by design, but now that the `enforceFocus` behavior is fixed, it will reset the scroll position to this part of the page, making it difficult to interact with the other examples. Happy to do in a separate PR if you prefer